src/backend/ipcHandlers/quests.ts

```diff
@@ -10,8 +10,14 @@ import {
 } from 'common/types'
 import { app, ipcMain } from 'electron'
 
-export async function getQuests(projectId?: string): Promise<Quest[]> {
-  let url = `${DEV_PORTAL_URL}api/v1/quests?questStatus=ACTIVE`
+async function fetchQuests({
+  projectId,
+  status
+}: {
+  projectId?: string
+  status: 'ACTIVE' | 'COMPLETED'
+}): Promise<Quest[]> {
+  let url = `${DEV_PORTAL_URL}api/v1/quests?questStatus=${status}`
   if (projectId) {
```

Comment:
Just a small nit pick but always better to use new Url() when adding parameters.